### ISSUE TITLE
Improve metric help messages

### ISSUE THREAD

**Issue by jvff** [linera-io/linera-protocol#1553]
# Motivation

When metric counters are created, they need a name string and a description string. The description is used as a help message, and should describe the metric in plain English. However, there are many metrics that use very similar strings for the names and the descriptions, which make the help message practically useless.

Here is an example: https://github.com/linera-io/linera-protocol/blob/589ddefa8cbafc731952bcab0b5c5c3150949111/linera-chain/src/chain.rs#L68-L69

# Proposal

Go through all created metrics and ensure they have helpful and clear descriptions.

**Comment by aman-at8** [comment]
hello @jvff, i have been going through linera's code for sometime now and I think I can work on this issue. I have also seen few places in the code where 'latency' is spelt as 'atency'  like [this](https://github.com/linera-io/linera-protocol/blob/bd98cdfb81bd368dd87ae382ca3e0a31954f01b0/linera-storage/src/chain_guards.rs#L171)

**Comment by SantiSjp** [comment]
@jvff this issue still available?
